### DISCUSSION TITLE
Inspiro light, setting max width of the site to 960px?

### DISCUSSION THREAD

**Discussion by Lufsen11** [wpzoom/inspiro#14] [Q&A]
Hello

Is there anybody who can give me advice on how to set up a max width of 960px to Inspiro -Lite?
I need this for a the presentation on large screens without effecting the smaller mobile screen.
I would like it to give an impression ofA4 or letter size when viewed on large screens,

Anybody? It would make my day to get some help with this.

**Comment by ciorici** [comment]
Hello @Lufsen11 

This is a difficult request, because we need to see what elements you have on your website.

Normally, you should just resize the browser window to the needed size of 960px, and the theme will adapt just fine, as it's a responsive theme.

A quick solution would be to add this CSS code in the Customizer > Additional CSS:

```
body {
    max-width: 960px;
    margin-left: auto;
    margin-right: auto;
}
```

However, it might not help if you have full-width sections created with the block editor or Elementor.


Another solution would be to add this code too:

```
body * {
    max-width: 960px !important;
    margin-left: auto !important;
    margin-right: auto !important;
}
```

**Comment by Lufsen11** [comment]
Thank you

I did the body, but I still have one container sticking out on the sides 
1.the hero image 
and
 2. The image bg above the footer. And I´m note skilled enough to figure this one out.

Thank you Oh so much for trying to help me solve this :-)

Berit Lindeberg-Åre Sweden

**Comment by Lufsen11** [comment]
And Oh I forgot, this is the page dev.avesaltus.se

**Comment by Lufsen11** [comment]
Hello again

Or at least it was now it is live.
https://www.avesaltus.se <https://www.avesaltus.se/>
And your code works well, 
although I still have the problem with the hero image growing out of proportion and off course getting blurry on a big screen.

But Thanks a lot :-)
Have a lovely day.
Berit-Sweden
